sdk/security/keyvault/TROUBLESHOOTING.md

```diff
@@ -6,32 +6,55 @@ common to these SDKs.
   * [HTTP 403 Errors](#http-403-errors)
     * [Operation Not Permitted](#operation-not-permitted)
     * [Access Denied to First Party Service](#access-denied-to-first-party-service)
-  * [HTTP 429: Too Many Requests](#http-429-too-many-requests)
+  * [Other Authentication Issues](#other-authentication-issues)
+    * [Incorrect Challenge Resource](#incorrect-challenge-resource)
 * [Other Service Errors](#other-service-errors)
+  * [HTTP 429: Too many requests](#http-429-too-many-requests)
 
-## Troubleshoot Authentication Issues
+## Authentication Errors
 
 ### HTTP 401 Errors
 
-HTTP 401 errors may indicate authentication problems.
+HTTP 401 errors may indicate authentication problems, but silent 401 errors are also an expected part of the Azure Key Vault authentication flow.
 
 #### Frequent HTTP 401 Errors in Logs
 
-Most often, this is expected. Azure Key Vault issues a challenge for initial requests that force authentication. You
-may see 401 responses in logs during application startup. If you don't see subsequent errors from the Key Vault client,
-these 401 responses are the expected authentication challenges.
+Most often, this is expected. Azure Key Vault issues a challenge for initial requests that force authentication. You may
+see these errors most often during application startup, but you may also see these periodically during the application's
+lifetime when authentication tokens are near expiration.
 
-#### Other Authentication Issues
+If you are not seeing subsequent exceptions from the Key Vault SDKs, authentication challenges are likely the cause. If
+you continuously see 401 errors without successful operations, there may be an issue with the authentication module
+that's being used. We recommend using the Azure SDK's [azidentity][azidentity] module for authentication.
 
-If you are using the `azidentity` module to authenticate Azure Key Vault clients, please see its
-[troubleshooting guide](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azidentity/TROUBLESHOOTING.md).
+#### AKV10032: Invalid Issuer
+
+You may see an error similar to:
+
+```text
+--------------------------------------------------------------------------------
+RESPONSE 401: 401 Unauthorized
+ERROR CODE: Unauthorized
+--------------------------------------------------------------------------------
+{
+  "error": {
+    "code": "Unauthorized",
+    "message": "AKV10032: Invalid issuer. Expected one of https://sts.windows.net/{tenant 1}/, found https://sts.windows.net/{tenant 2}/."
+  }
+}
+```
+
+This is most often caused by being logged into a different tenant than the Key Vault authenticates.
+See our [DefaultAzureCredential][DefaultAzureCredential] documentation to see the order credentials are read. You may be logged into a different
+tenant for one credential that gets read before another credential. For example, you might be logged into Visual Studio
+under the wrong tenant even though you're logged into the Azure CLI under the right tenant.
```

Comment:
The Key Vault doesn't authenticate a tenant; it resides in a tenant. Also, we don't share authentication with Visual Studio/Code, and DefaultAzureCredential isn't necessarily relevant here. Taking a step back though, this particular error will also go away once clients specify the Vault's tenant in their authentication requests, so if we're going to ship that feature soon, we don't need to document this error.